Source/WebCore/animation/AnimationEffectTiming.h

```diff
@@ -56,9 +58,17 @@ struct AnimationEffectTiming {
     CSSNumberishTime activeDuration { 0_s };
     CSSNumberishTime endTime { 0_s };
 
+    struct ResolutionData {
+        std::optional<CSSNumberishTime> timelineTime;
```

Comment:
All these optionals make this pretty space-inefficient. Consider `Markable<>`.

Reply:
Adding it to my todo list of cleanups.